cmd/nigiri/resources/docker-compose.yml

```diff
@@ -119,7 +119,7 @@ services:
     environment:
       API_URL: http://localhost:3001
     ports:
-      - 5001:5000
+      - 4401:5000
```

Comment:
Still, this is used in many tutorials already. Best to give a different one to new services introduced if possible